guarddog/cli.py

```diff
@@ -78,18 +78,18 @@ def _verify(path, rules, exclude_rules, reporter, exit_non_zero_on_finding, ecos
     for result in results:
         identifier = result['dependency'] if result['version'] is None \
             else f"{result['dependency']} version {result['version']}"
-        if reporter != "json":
+        if output_format != "json":
```

Comment:
we should probably make this generic when we'll add more types, I guess the suggestion below should do it? (untested)

```suggestion
        if output_format is None:
```